### PR TITLE
Adding Test::Mojo::message_* tests.

### PR DESCRIPTION
**Pull request by sergiotarxz**
### Summary
Adding tests for the Test::Mojo::message_* methods.

### Motivation
Issue #1488.

### References
#1488

### CODE REVIEW

File: t/test/mojo.t [hunk 1]
```diff
@@ -5,6 +5,10 @@ use Test::Mojo;
 use Mojolicious::Lite;
 
 any '/' => {text => 'Hello Test!'};
+websocket '/websocket', => sub {
```
Comment (kraih) [inline t/test/mojo.t:8]: Missing blank line above.

File: t/test/mojo.t [hunk 2]
```diff
@@ -5,6 +5,10 @@ use Test::Mojo;
...
+  my ($c) = @_;
```
Comment (kraih) [inline t/test/mojo.t:9]: We use the `= shift;` convention everywhere else.

File: t/test/mojo.t (hunk 1 above)
Comment (kraih) [reply t/test/mojo.t:8]: I assume `, =>` is a typo.

File: t/test/mojo.t [hunk 3]
```diff
@@ -217,4 +221,45 @@ subtest 'header_unlike' => sub {
   is_deeply \@args, ['unlike', 'text/html;charset=UTF-8', qr/image\/png/, 'some description'], 'right result';
 };
 
+subtest 'message_is' => sub {
+  $t->websocket_ok('/websocket')->message_ok->message_is('testing message');
+  is_deeply \@args, ['is', ('testing message') x 2, 'exact match for message'], 'message_is succeded.';
```
Comment (kraih) [inline t/test/mojo.t:226]: Don't get fancy with the `x 2`, we prefer readability in tests, even if it ends up a little more verbose.

Comment (kraih) [reply t/test/mojo.t:226]: And please use test descriptions consistent with other tests.

File: t/test/mojo.t (hunk 1 above)
Comment (sergiotarxz) [reply t/test/mojo.t]: Done.

File: t/test/mojo.t (hunk 2 above)
Comment (sergiotarxz) [reply t/test/mojo.t]: Done.

File: t/test/mojo.t (hunk 3 above)
Comment (sergiotarxz) [reply t/test/mojo.t]: Done

**Comment by sergiotarxz** [comment]
I think it is not my fault these two test fail:

```
@github-actions
linux / perl (5.22) (pull_request) Failing after 2m
Required
[Details](https://github.com/mojolicious/mojo/actions/runs/7862274388/job/21451556660?pr=2153)
@github-actions
linux / perl (5.30) (pull_request) Failing after 2m
```
I think it failed to download extutils::makemaker.

**Review by kraih — APPROVED** [review]

**Pull request merged**